warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -1127,7 +1127,9 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
       }      
     } catch (Throwable t) { // Catch everything else this handler could return 200 on a OOM exception
       if (!response.isCommitted()) {
-        String msg = "Error when updating data: " + ThrowableUtils.getErrorMessage(t);
+        String prefix = "Error when updating data: ";
+        // HTTP reason in Jetty is 1024 chars max
+        String msg = prefix + ThrowableUtils.getErrorMessage(t, 1024 - prefix.length());
```

Comment:
Could this 1024 be defined as a constant somewhere, with an accompanying comment so its meaning is clear when reading the code.

Reply:
Or better yet, use Jetty's own constant as the default value of that new constant.